package.json

```diff
@@ -56,7 +56,7 @@
           "circulation-storage.loan-policies.collection.get",
           "circulation.end-patron-action-session.post",
           "configuration.entries.collection.get",
-          "notes.collection.get.by.status"
+          "note.links.collection.get"
```

Comment:
If this permission is only available with the `inventory: 14.0` interface, please update it to `14.0`. We also add `*BREAKING*` to the changelog description when this is a breaking change.

Reply:
This permission is only available in `notes: 4.0`. But I cannot see this interface in `okapiInterfaces` section.

Reply:
Hi @Dmitriy-Litvinenko, the module doesn't have a `notes` interface. Does it make a request for notes? Are notes permissions still required for for the module?

Reply:
The interfcace added to dependencies.